app/styles/play/campaign-view.sass

```diff
@@ -938,13 +938,6 @@ $gameControlMargin: 30px
     z-index: 1
     width: 193px
 
-  #videos-button
-    position: absolute
-    top: calc(1% + 128px)
-    left: 1%
-    z-index: 1
-    width: 193px
-
```

Comment:
We don't need this CSS any more?

Reply:
We don't. This hard codes the location of the videos button beneath the `.under-logo` location. I lifted that into a container class, and now the buttons can be listed out within the div container. Thus we don't have to hardcode the location of a button based on the number of buttons above it.